helloworld-simple-project/src/main/java/org/acme/GreetingResource.java

```diff
@@ -0,0 +1,16 @@
+package org.acme;
```

Comment:
This file does not look like a serverless function? Is this what the kn create command created by default?

Reply:
So yes I have created the function using kn create and did the changes in the business logic.

Reply:
you might also want to change the package name to sth like `com.redhat.training.examples`